azure/services/loadbalancers/spec.go

```diff
@@ -289,7 +289,7 @@ func getProbes(lbSpec LBSpec) []*armnetwork.Probe {
 					Port:              ptr.To[int32](lbSpec.APIServerPort),
 					RequestPath:       ptr.To(httpsProbeRequestPath),
 					IntervalInSeconds: ptr.To[int32](15),
-					NumberOfProbes:    ptr.To[int32](4),
+					ProbeThreshold:    ptr.To[int32](1),
```

Comment:
Is there a reason why the probe threshold is being changed from 4 to 1, assuming the field works the same as number of probes?